### PR TITLE
prepare device message

### PR DESCRIPTION
**Pull request by r10s**
this is the device message, focusing on email-compat, but trying to not raise too many expectations:

Screen Shot 2021-05-04 at 10 41 28

blog-post with much more details ~~is up to come~~ drafted at https://github.com/deltachat/deltachat-pages/pull/439, so please do not merge yet.

cmp. https://github.com/deltachat/deltachat-ios/pull/1195

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
 # Delta Chat Android Changelog
 
-## UNRELEASED
+## v1.20.0
 
+* improved accessibility and screen reader support
+* use the same emoji style everywhere across the app
```
Comment (adbenitez) [inline CHANGELOG.md:6]:
```suggestion
* use the same emoji style everywhere across the app
* allow to select and copy text from "message details" and error dialogs
```

**Review by adbenitez — APPROVED** [review]
lgtm, but the device message is missing a lot of things that was added/improved and that justify the delay in releases

(EDIT: never mind, there is a link to blog post)

**Review by hpk42 — APPROVED** [review]
nice

**Comment by r10s** [comment]
i'd like to concentrate on the changes wrt email-compatibility; this is the hugest part. and that way, we can use the same device message on ios/desktop (this is not strictly needed, however, makes it easier for them to release)

details can go to the blog-post.

**Pull request merged**